Gemfile.lock

```diff
@@ -1,21 +1,21 @@
 PATH
   remote: .
   specs:
-    faraday-restrict-ip-addresses (0.0.2)
-      faraday (>= 0.8, < 0.9)
+    faraday-restrict-ip-addresses (0.1.2)
```

Comment:
The new version os 0.1.3, but this is 0.1.2. I think you need to run `bundle install` again and push the result.